JS/script.js

```diff
@@ -25,4 +65,30 @@ closeMobileMenu.forEach((button) => {
     const mobileMenu = button.closest('.mobile-menu');
     closeMenu(mobileMenu);
   });
-});
\ No newline at end of file
+});
+projectDetails.forEach((button) => {
+  const test = button;
+  button.addEventListener('click', () => {
+    const idModal = test.getAttribute('id');
+
+    const modalWindow = document.querySelector('.project-details-modal');
+    modalWindow.classList.add('showModal', 'animate__bounceInDown');
+    mainTag.classList.add('blur');
+    const h3Test = modalWindow.querySelector('h3');
+    const paragraph = modalWindow.querySelector('.work-description p');
+    const imageNode = modalWindow.querySelector('.work-img img');
+    const githubButton = modalWindow.querySelector('.github');
+    const arrowCircleButton = modalWindow.querySelector('.arrow_circle');
+    h3Test.innerHTML = projects[idModal].name;
+    paragraph.innerHTML = projects[idModal].descrition;
+    imageNode.src = projects[idModal].image;
+    githubButton.src = projects[idModal].imageGit;
+    arrowCircleButton.src = projects[idModal].imageLive;
+    const closeModalMobile = modalWindow.querySelector('[data-close-button]');
+
+    closeModalMobile.addEventListener('click', () => {
+      modalWindow.classList.remove('showModal', 'animate__bounceInDown');
+      mainTag.classList.remove('blur');
+    });
+  });
+});
```

Comment:
[OPTIONAL] I think we could wrap all of our JS code in a `main` function. And the call `main` once the web app is finished loading. This will ensure that it works correctly. Imagine that the script loads, but maybe some of the elements on the document aren't ready yet. Then it could fail to select these.

Start the script running once the page has finished loading like this: `window.onload = main`

Reply:
I will work on that. Thank you